test/ExecutionTests.jl

```diff
@@ -148,12 +148,28 @@ tune!(b_pass)
 # warmup #
 ###########
 
-p = params(warmup(@benchmarkable sin(1)))
+@test_deprecated warmup(@benchmarkable sin(1))
+
+is_warm = false
+function needs_warm()
+    global is_warm
+    if is_warm
+        sleep(0.1)
+    else
+        sleep(2)
+        is_warm = true
+    end
+end
+
+w = @benchmarkable needs_warm()
+w.params.seconds = 1
+tune!(w)
```

Comment:
Is there a way to check that `tune!` warms up too?

Reply:
Sure! I can check that.